server/repository/src/db_diesel/program_requisition/program_requisition_order_type_row.rs

```diff
@@ -0,0 +1,72 @@
+use super::{
+    program_requisition_order_type_row::program_requisition_order_type::dsl as program_requisition_order_type_dsl,
+    program_requisition_settings_row::program_requisition_settings,
+};
+
+use crate::{repository_error::RepositoryError, StorageConnection};
+
+use diesel::prelude::*;
+
+table! {
+    program_requisition_order_type (id) {
+        id -> Text,
+        program_requisition_settings_id -> Text,
+        name -> Text,
+        threshold_mos -> Double,
+        max_mos -> Double,
+        max_order_per_period -> Double,
+    }
+}
+
+joinable!(program_requisition_order_type -> program_requisition_settings (program_requisition_settings_id));
+
+#[derive(Clone, Queryable, Insertable, AsChangeset, Debug, PartialEq, Default)]
+#[table_name = "program_requisition_order_type"]
+pub struct ProgramRequisitionOrderTypeRow {
+    pub id: String,
+    pub program_requisition_settings_id: String,
+    pub name: String,
+    pub threshold_mos: f64,
+    pub max_mos: f64,
+    pub max_order_per_period: f64,
```

Comment:
Does it makes sense for this to be int?
mSupply doesn't accept non-integer values

Reply:
Do you mean it should be an int (for the max_order_per_period)? 
MOS is float in central as well ....

Reply:
Yes, `max_order_per_period`, basically I figure it doesn't make sense to have half an order?
Not too worried, but thought I'd mention it. @andreievg any thoughts.?

Reply:
ohh yeah that one should just be integer, thanks